src/Microsoft.Unity.Analyzers.Tests/ContextMenuSuppressorTests.cs

```diff
@@ -64,11 +63,11 @@ public async Task ContextMenuItemUnusedSuppressed()
 class Menu : MonoBehaviour
 {
     [ContextMenuItem(""Foo"", ""Bar"")]
-    private string contextMenuString = """";
+    readonly string contextMenuString = """"; // we only use readonly here for testing purposes, suppressor are tested unitarily
```

Comment:
Would be nice to add to the comment the root cause for the change, something this like: « We're using readonly to avoid triggering a `CSXXXX, fields can not be bla bla,` » Right now I understand why we want to avoid a warning or refactoring suggestion, I just don't know which one.

Reply:
Done.